webrtc.html

```diff
@@ -12001,8 +12001,8 @@ <h3 id="sec-constraints">Constraints</h3>
       default values for attributes that would be left uninitialized without
       the init dictionary present.</li>
       <li>InvalidMediaStreamTrackError changed to InvalidParameter.</li>
-      <li>Fire NetworkError when the data transport is closed with an
-      error.</li>
+      <li>Fire the <code><a>RTCError</a></code> event
+      when the data transport is closed with an error.</li>
```

Comment:
@taylor-b This relates to an error in the data channel transport,  such as might be communicated in an SCTP ERROR chunk.  The Cause Code values are listed here: http://www.iana.org/assignments/sctp-parameters/sctp-parameters.xhtml#sctp-parameters-24

The ERROR chunk also has a Cause-Specific-Information field.  So perhaps RTCError.errorDetail contains the Cause Code and RTCError.message contains the Cause-Specific-Information?